src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -15,8 +15,9 @@ public class DefaultHubLifetimeManager<THub> : HubLifetimeManager<THub>
         private long _nextInvocationId = 0;
         private readonly ConnectionList _connections = new ConnectionList();
 
-        public override Task AddGroupAsync(ConnectionContext connection, string groupName)
+        public override Task AddGroupAsync(string connectionId, string groupName)
         {
+            var connection = _connections[connectionId];
```

Comment:
The `ConnectionList` type will return null if it can't resolve the connectionId that was passed in. I should add a null check here.